### PR TITLE
[Guide] Secure Introduction - Update 

### PR DESCRIPTION
**Pull request by yhyakuna**
Adding **Vault Agent** to this guide. 

screenshot

### CODE REVIEW

File: website/source/guides/identity/secure-intro.html.md [hunk 1]
```diff
@@ -133,6 +137,37 @@ this model can be applied regardless of where the applications are running.
   - [Cubbyhole Response Wrapping](/guides/secret-mgmt/cubbyhole.html)
 
 
+### Vault Agent
+
+Vault agent allows the implementation of the [platform integration](#platform-integration)
```
Comment (armon) [inline website/source/guides/identity/secure-intro.html.md]: I would rephrase, something like "Vault agent automates the workflow of client login and token refresh, and can be used with either platform integration or trusted orchestrator approaches."

File: website/source/guides/identity/secure-intro.html.md [hunk 2]
```diff
@@ -133,6 +137,37 @@ this model can be applied regardless of where the applications are running.
...
+where the token to be written.
+
+When the agent is started, it will attempt to acquire a Vault token using the
+auth method specified in the agent configuration file.  On successful
+authentication, the resulting token is written to the sink locations.
+Optionally, this token can be response-wrapped or encrypted. Whenever the
+current token value changes, the agent writes to the sinks. If authentication
+fails, the agent waits for a while and then retry.
+
+The client can simply retrieve the token from the sink and connect to Vault
```
Comment (armon) [inline website/source/guides/identity/secure-intro.html.md]: I would add: "This simplifies client integration, since the Vault agent handles the login and refresh logic."

Comment (yhyakuna) [reply website/source/guides/identity/secure-intro.html.md]: Thank you!  Incorporated in [147540b](https://github.com/hashicorp/vault/pull/5323/commits/147540b7cda18f2ee14aa4ca3b758e44b6e4d8d8)

File: website/source/guides/identity/secure-intro.html.md (hunk 1 above)
Comment (yhyakuna) [reply website/source/guides/identity/secure-intro.html.md]: Thank you!  Incorporated in [147540b](https://github.com/hashicorp/vault/pull/5323/commits/147540b7cda18f2ee14aa4ca3b758e44b6e4d8d8)
